llvm/test/Transforms/LoopVectorize/X86/vplan-noinbounds-gep.ll

```diff
@@ -0,0 +1,104 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 3
+; RUN: opt -passes=loop-vectorize -force-vector-width=4 -S %s | FileCheck %s
+
+target datalayout = "e-m:e-p270:32:32-p271:32:32-p272:64:64-i64:64-i128:128-f80:128-n8:16:32:64-S128"
+target triple = "x86_64-unknown-linux-gnu"
+
+; Function Attrs: nofree norecurse nosync nounwind memory(none)
+define i1 @fn(ptr %nno) local_unnamed_addr #0 {
```

Comment:
Would be good to add a comment sayin what this tests, e.g. something like `FIXME:  GEP flags on GEPs for reverse vector pointer need to be dropped when folding the tail.`